lib/wrapInitialConfig.js

```diff
@@ -10,6 +10,15 @@ module.exports = (api) => {
     'hash-module-ids',
   ];
 
+  if (options && options.allowedPlugins) {
+    allowedPlugins = options.allowedPlugins.reduce((allPlugins, plugin) => {
+      if (allPlugins.indexOf(plugin) < 0) {
+        allPlugins.push(plugin);
+      }
+      return allPlugins;
+    }, allowedPlugins);
+  }
+
   existingPlugins.forEach((plugin) => {
     if (!allowedPlugins.includes(plugin)) {
```

Comment:
You can do something like `if (!allowedPlugins.includes(plugin) && !options.allowedPlugins.includes(plugin)) {`

Reply:
Just wanted to extend existing array in case we need to use it in future. if this is not necessary, I can do as you say